packs/feat-effects/effect-share-rage.json

```diff
@@ -45,10 +45,24 @@
                 "value": 0.5
             },
             {
-                "key": "FlatModifier",
-                "label": "PF2E.TraitRage",
-                "selector": "ac",
-                "value": -1
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "flags.pf2e.rageTempHP",
+                "phase": "beforeDerived",
+                "predicate": [
+                    {
+                        "nor": [
+                            "self:type:familiar",
+                            "self:type:hazard",
+                            "self:type:vehicle"
+                        ]
+                    }
+                ],
+                "value": "@actor.level + @actor.abilities.con.mod"
```

Comment:
I think technically this should just be in the temp HP rule and skip the AE like here. If this went on an ally that overrides rage HP for their own thing they should get the benefits as described in the barbarian rage, not their particular flavor. Like a summoned giant badger shouldn't get 0 temp HP from this.

Reply:
I was thinking that it should be affected by the Rage temp HP immunity effect, since allies would also not be able to benefit from them again, as well as supporting Contagious Rage in the future, but maybe we'll cross that bridge when we get there.

Reply:
It's an effect, so at least no migration will ever be needed.